CMakeLists.txt

```diff
@@ -34,6 +38,10 @@ if (NOT DEVICE)
     # set(BLE_LIB EFR32BG13P)
 endif ()
 
+if (NOT JLINK_DEVICE)
```

Comment:
could we move this to `jlink.cmake` so the change is non-breaking for existing consumers?

(ie. unless they override `JLINK_DEVICE` it will default to `DEVICE` without a change to their project `CMakeLists.txt`)